blimp/source/docs/servo-rc-setup.rst

```diff
@@ -0,0 +1,24 @@
+.. _servo-rc-setup:
+
+==============
+SERVO/RC SETUP
+==============
+
+The flapping fin Blimp requires for servos for movement and altitude control. Therefore, four outputs on the autopilot will need to be assigned to the Front,Rear, Right, and Left side servos. This is accomplished by setting the ``SERVOx_FUNCTION`` for each output connected to these servos as follows:
+
+==============      ===============
+SERVO POSITION      SERVOx_FUNCTION
+==============      ===============
+REAR                  Motor 33
+FRONT                 Motor 34
+RIGHT                 Motor 35
+LEFT                  Motor 36
+==============      ===============
+
+
+ARMING SWITCH
+=============
+
+While it is possible to use :ref:`ARMING_RUDDER<ARMING_RUDDER>` = 2, to allow arming and disarming via the throttle and rudder sticks, unless this is done in the LAND flight mode, undesired movement of the Blimp will occur.
```

Comment:
The main problem is that if ARMING_RUDDER is set to 2, then trying to yaw while flying is likely to result in the blimp disarming mid-air - because it's problematic to do land-detection for a blimp (since it never really "lands"), so at the moment, rudder disarming would still be active in-flight if that parameter is set.

Reply:
changed